cypress/integration/search_for_property.spec.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types="cypress" />
+
+beforeEach(() => {
+  // Use properties fixture as mock stubbed data
+  cy.fixture('properties/properties.json').as('propertiesList')
+  cy.fixture('properties/property.json').as('property')
+  cy.server()
+})
+
+describe('Search for property', () => {
+  it('Search for property by postcode', () => {
```

Comment:
Should we split out part of this into a spec for searching by address?

Reply:
Yeah although maybe just the initial search as there will be duplication from above when going into the actual property. Unless that is OK?

Reply:
Yes I was thinking along same lines as you, and minimal duplication - just check the address search is possible and has the check for number of results, but no more steps than that.